supertokens_python/recipe/jwt/interfaces.py

```diff
@@ -40,8 +40,9 @@ class CreateJwtResultUnsupportedAlgorithm:
 
 
 class GetJWKSResult:
-    def __init__(self, keys: List[JsonWebKey]):
+    def __init__(self, keys: List[JsonWebKey], validity_in_secs: Optional[int]):
```

Comment:
validity_in_secs does not need to be optional

Reply:
Not doing this based on our discussion on slack.